features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/item-info/js/load-tabs.js

```diff
@@ -77,7 +77,7 @@ $(document).ready(function() {
             var clickedTab = e.target.href.split('#')[1];
             ////////////// edit tab
             pushDataForTabs(clickedTab);
-            $.cookie("selectedTab",clickedTab);
+            $.cookie("selectedTab", clickedTab,  {path: "/"});
```

Comment:
These are minor issues but please make sure to be consistent. There are 2 spaces between ,  {path: "/"}